demo/index.js

```diff
@@ -77,13 +77,8 @@ api.route('/users/:userId', userSchema, {
 });
 
 // Defines the route that manages a collection of users
-api.route('/users', userCollectionSchema, {
-    get: function(request, connection) {
-        var users = mockUserEmails.map(function(user) {
-            return new Symlink('/users/' + user.userId);
-        });
-        return new Collection('/users', users, 1000);
-    },
+api.route(['/users', 'limit=10&offset=0&search'], userCollectionSchema, {
```

Comment:
do we need this search parameter in the resource? we're not doing anything with it in getUsers... may just confuse people